app/assets/stylesheets/config/_import.scss

```diff
@@ -36,6 +36,7 @@ $colors: (
 //
 // List of breakpoints for our responsiveness.
 
+// If you modify these breakpoints, ensure you update them in @components/useMediaQuery.js
```

Comment:
Once we move to Tailwind, this won't be an issue as we can import the breakpoints in JS land.